proto-google-cloud-bigtable-v2/src/main/java/com/google/bigtable/v2/ReadChangeStreamRequestOrBuilder.java

```diff
@@ -110,7 +110,7 @@
    *
    * @return The partition.
    */
-  StreamPartition getPartition();
+  com.google.bigtable.v2.StreamPartition getPartition();
   /**
```

Comment:
Curious: why are these types fully qualified now?

Reply:
I'm not sure.. But this is how it should be.